ui/k-info-panel/edit_window.kytos

```diff
@@ -0,0 +1,718 @@
+                        "id": this.chosen_links[i]
+                    }
+                    // Add the JSON link to the list of items.
+                    jsonItems.push(link)
+                }
+                // For every chosen switch
+                for(i = 0; i < this.chosen_switches.length; i++) {
+                    // If the switch is not a string
+                    if(typeof(this.chosen_switches[i]) != "string") {
+                        // Skip it.
+                        continue
+                    }
+                    // Otherwise, add the switch id to the list of items.
+                    jsonItems.push(this.chosen_switches[i])
+                }
+                // For every chosen interface
+                for(i = 0; i < this.chosen_interfaces.length; i++) {
+                    // If the interface is not a string
+                    if(typeof(this.chosen_interfaces[i]) != "string") {
+                        // Skip it.
+                        continue
+                    }
+                    // Otherwise, construct a JSON object with the interface id.
+                    interface = {
+                        "interface_id": this.chosen_interfaces[i]
+                    }
+                    // Add the JSON interface to the list of items.
+                    jsonItems.push(interface)
+                }
+
+                // Update the maitenance window with the new values.
+                var request = $.ajax({
+                    url: this.$kytos_server_api + "kytos/maintenance/" + _this.window_data.Id,
+                    type: "PATCH",
+                    data: JSON.stringify({
+                        "id": _this.window_data.Id,
+                        "start": _this.window_data.Start,
+                        "end": _this.window_data.End,
+                        "description": _this.window_data.Description,
+                        "items": jsonItems,
+                        "status": status
+                    }),
+                    dataType: "json",
+                    contentType: "application/json"
+                })
+                request.done( function(data) {
+                    let notification = {
+                        title: 'Window "' + _this.window_data.Id + '" updated.',
+                        description: ''
+                    }
+                    // Notify on success.
+                    _this.$kytos.$emit("setNotification", notification)
+                    // Refresh the window.
+                    _this.resetItemSelection()
+                    _this.loadMaintenanceWindow(_this.window_data.Id)
+                })
+                request.fail(function(jqXHR, status, error ) {
+                    let notification = {
+                        title: 'Updated Window failed.',
+                        description: 'Error updating window "' + _this.window_data.Id + '". ' + _this.capitalize(status) + ': ' +  JSON.parse(jqXHR.responseText).response
```

Comment:
@Aduan002, currently when it returns a 400, the rendered message shows an undefined error in the notification window:
```
Updated Window failed.
Error updating window "some_mw". Error: undefined
```
However, the backend provides the reason why in the response. Could you format this error message to include the description if it's set?

```
{"code":400,"description":"Start in the past not allowed.","name":"Bad Request"}
```